eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -20,7 +20,7 @@
     <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.CSharp/*3.3.1*" />
     <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.CSharp.Workspaces/*3.3.1*" />
     <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.Workspaces.Common/*3.3.1*" />
-    <UsagePattern IdentityGlob="System.Composition/*1.0.31*" />
+    <UsagePattern IdentityGlob="System.Composition/*7.0.0*" />
```

Comment:
This seems odd. Who is using this?

Reply:
Yes, we need to understand where the dependency is coming from. Modifying the prebuilt baseline without understanding the nature of the dependency can potentially break source-build for the whole product.

Reply:
This is a reaction to the M.C.A version upgrade. Upon further analysis, it looks like the correct reaction might be to add entries for this package to our Versions.Details.xml as seen in https://github.com/dotnet/roslyn/pull/68371. Does that seem right, @mthalman?

Reply:
Yep, that should do it. Then this `System.Composition` entry should be removed from the baseline file.

Reply:
Sure -- I can pull these changes out into a separate PR.

Reply:
See https://github.com/dotnet/aspnetcore/pull/49245